tensorboard/webapp/widgets/data_table/filter_dialog.scss

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@use '@angular/material' as mat;
+@import 'tensorboard/webapp/theme/tb_theme';
+
+.filter-dialog {
+  padding: 8px;
+  border-radius: 4px;
+  border: 1px solid;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-color: mat.get-color-from-palette($tb-foreground, border);
+  background-color: mat.get-color-from-palette($tb-background, background);
+
+  @include tb-dark-theme {
+    border-color: mat.get-color-from-palette($tb-dark-foreground, border);
+    background-color: mat.get-color-from-palette(
+      $tb-dark-background,
+      'background'
+    );
+  }
+
+  .discrete-filters-area {
+    max-height: 100px;
+    overflow-y: auto;
+  }
+
+  .no-matches {
+    // 24px is the width of the checkbox so the text should match the
```

Comment:
For what it's worth, I played with it a bit and it didn't seem like the "No matching values" text ever "matched the indentation of the selectable filters" for any definition of "matched".

![image](https://github.com/tensorflow/tensorboard/assets/17152369/dbe0ef17-99aa-48cb-82cc-aa7d5a6bc008)

![image](https://github.com/tensorflow/tensorboard/assets/17152369/4eeb1af6-6c4a-493a-a97e-bd9c245e5621)

Reply:
I don't think I checked after changing to use the new components. Apparently the padding is now `12px`.